.travis.yml

```diff
@@ -21,6 +21,10 @@ before_script:
 script:
   - yarn lint
   - AST_COMPARE=1 yarn test -- --runInBand
+  # A basic test to ensure prettier runs without devDependencies
+  - rm -rf node_modules
```

Comment:
This busts travis cache which greatly speeds up tests.

Can you instead use npm pack and npm install that file in the test that builds the prod version?